kanban-server/routes/board.route.js

```diff
@@ -1,19 +1,14 @@
-const express = require('express');
-const BoardController = require('../controllers/board.controller');
-const verifyAuth = require('../middlewares/auth');
-const { boardNameValidator } = require('../validators/board.validator');
+const express = require("express");
+const BoardController = require("../controllers/board.controller");
+const verifyAuth = require("../middlewares/auth");
+const { boardNameValidator } = require("../validators/board.validator");
 
 const router = express.Router();
 
-router.use('/board', router);
+router.use("/board", router);
 
-router.post('/create', boardNameValidator, verifyAuth, BoardController.create);
+router.post("/create", boardNameValidator, verifyAuth, BoardController.create);
 
-router.patch(
-	'/update/:id',
-	boardNameValidator,
-	verifyAuth,
-	BoardController.update,
-);
+router.put("/:id", boardNameValidator, verifyAuth, BoardController.update);
```

Comment:
Can we use patch here? as we're updating the board name only... let me know your suggestions.. @hemanthsharma13

Reply:
Yea, but patch can override the existing schema sometimes. So put is prefered but we can discuss on it